source/reference/access-mdb-shell-help.txt

```diff
@@ -0,0 +1,241 @@
+.. _mdb-shell-help:
+
+===========================
+Access the |mdb-shell| Help
+===========================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+``mongosh`` provides additional help information in its "online"
+help system. This document provides an overview of accessing this help
+information. 
+
+.. _mdb-shell-help-command-line:
+
+Command Line Help
+-----------------
+
+To see the list of options and help for starting ``mongosh``, use the
+``--help`` option from the command line:
+
+.. code-block:: sh
+
+   mongosh --help
+
+Shell Help
+----------
+
+To see the list of help, in ``mongosh``, type ``help``:
```

Comment:
"list of help" ? seems like an incorrect or missing word here.

Reply:
updated